### ISSUE TITLE
ультра объемы в индикаторе Total Delta 

### ISSUE THREAD

**Issue by Dimitrian** [SBProX/SBProX-Dev#1863]
Иногда появляются ультраобъемы без конца и края (сколько не тяни масштаб). Из-за этого все более мелкие объемы становятся едва различимы и неудобны для анализа.
Предложение: Искусственно ограничивать высоту ультраобъемов до соизмеримых значений в рамках видимого на экране диапазона и помечать их допустим другим цветом. 
![проблемный ультраобъем](https://user-images.githubusercontent.com/77572690/107225829-5dcfc400-6a2a-11eb-9187-139bc6cba9e7.jpg)

**Comment by ProgramGun** [comment]
Да, поддерживаю, какую-то бы галку что бы их фильтровать, так как в экран не влазят.
Еще от себя очень прошу добавить сколько дельта в % на свечу (не только цифры)

**Comment by Dimitrian** [comment]
Вопрос как я понимаю еще не рассмотрен. А проблема весьма неприятная, вот сегодня снова "засада".
![ультра объем](https://user-images.githubusercontent.com/77572690/130592699-41bc7c2e-9ef0-4548-9570-15c8cd327fe5.jpg)
